bitnami/external-dns/Chart.yaml

```
@@ -28,4 +28,4 @@ maintainers:
 name: external-dns
 sources:
 - https://github.com/bitnami/charts/tree/main/bitnami/external-dns
-version: 6.26.5
+version: 6.26.6
```

Given this is a feature addition, we should update the version to a new minor.

```suggestion
version: 6.27.0
```

---

check!